src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -483,8 +483,13 @@ function MoneyRequestConfirmationList({
             return;
         }
 
+        if (transaction?.splitShares[currentUserPersonalDetails.accountID] && (transaction?.splitShares[currentUserPersonalDetails.accountID]?.amount ?? 0) === 0) {
```

Comment:
@Krishna2323 Please add a comment here that states this change was to prevent setting your own split to 0.